LunaMary/src/Ask.java

```diff
@@ -0,0 +1,42 @@
+import java.util.List;
+import java.util.Scanner;
+
+
+class Ask {
+
+    List<OptionAsk> optionAsks;
+    private String ask;
+    private String sigla;
+
+    public Ask(List<OptionAsk> optionAsks,
+               String ask, String sigla) {
+        this.optionAsks = optionAsks;
```

Comment:
When working with collections is not a great idea to pass them as parameters because copying collections can cause errors and performance issues.
I'll suggest to add methods that will wrap the use of your collection, for example:
public String addQuestionOption(QuestionOption option) {
    this.questionOptions.add(option);
}
You can do this so in the constructor you will not receive a list as a parameter and you will only initialize your list:
this.questionOptions = new ArrayList<>();